### PR TITLE
Upgrading to ReactNative 0.53.3

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
The `Android` app compiles and runs successfully. I haven't updated the `iOS Podspecs` yet. Will update that and verify.

File: ReactNativeTemplate/package.json [hunk 1]
```diff
@@ -11,10 +11,11 @@
         "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev"
     },
     "dependencies": {
-        "react": "16.0.0",
-        "react-native": "0.50.4",
+        "react": "16.2.0",
```
Comment (wmathurin) [inline ReactNativeTemplate/package.json]: I wonder if we even need these here since react-native-force should pull in anything it needs.

Comment (bhariharan) [reply ReactNativeTemplate/package.json]: It complains: `UNMET PEER DEPENDENCY`.

Comment (wmathurin) [reply ReactNativeTemplate/package.json]: Hmm that sounds familiar - maybe I did try that before.

**Comment by bhariharan** [comment]
The `iOS` app compiles and runs successfully as well.

**Pull request merged**